pyproject.toml

```diff
@@ -239,8 +239,19 @@ log_date_format = "%Y-%m-%d %H:%M:%S"
 asyncio_mode = "auto"
 
 [tool.ruff]
+required-version = "0.0.231"
```

Comment:
It looks like this needs to match running version
> error: Required version `0.0.231` does not match the running version `0.0.235`

Reply:
Oh boy, that sucks, going to remove it in that case.